rails-questions.md

```diff
@@ -2,26 +2,31 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
-
+rails new sample_app -d postgresql -T
 
 2. Explain each section of the following route:  get '/library/:id' => 'book#show'
 
-get -
-library -
-:id -
-book -
-show -
+get - The http verb we are sending to the server through the url.
+library - The url itself.
+:id - the parameter?? 
+book - The name of the controller
+show - The name of the method that we put in the controller 
 
 3. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
 
-
+rails generate migration migration_name
```

Comment:
you'll need 2 more steps here:
1. edit the migration file to reflect the changes you're making to your db
2. run rails db:migrate 

:)